Sources/SwiftGRPCNIOTestData/GRPCSwiftCertificate.swift

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright 2018, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import Foundation
+import NIOSSL
+
+/// Wraps `NIOSSLCertificate` to provide the certificate common name and expiry date.
+public struct GRPCSwiftCertificate {
```

Comment:
nit: is there a way to make it even clearer naming-wise that these are just intended as sample certificates? E.g. `SampleCertificate`? Just an idea.

Similarly, the `TestData` module name feels a bit weird to me. How about `SwiftGRPCNIO{Ex,S}ampleData`? Still not yet sure myself how much better these would be, let's discuss this.

Reply:
`SampleCertificate` is better. `SwiftGRPCNIOSampleData` works for me too, I wasn't keen on the naming before!